build/IPDK_Container/README

```diff
@@ -32,14 +32,16 @@ Pre-requisites to run IPDK container:
 	- systemctl daemon-reload
 	- systemctl restart docker
 
-Note: Build and run your IPDK container from ‘build/IPDK_Container’ directory
- in ‘https://github.com/ipdk-io/ipdk’ repository.
+Note: Build and run your IPDK Fedora33 container from ‘build/IPDK_Container’
+ directory or Ubuntu20.04 container from ‘build/IPDK_Container/Ubuntu20.04’
+ directory in ‘https://github.com/ipdk-io/ipdk’ repository.
 
 Commands to bring-up container and run OvS with P4:
 ---------------------------------------------------
 1) Edit the IPDK_Container/docker.env and IPDK_Container/start_p4ovs.sh files 
    to add your proxy
 2) Run - make build-nc - To build a docker image.
+   Use make build-nc-proxy to be used while running behind proxy.
```

Comment:
Looks like this option is taken care for only Ubuntu 20.04 containers. README file which modified for common.  If possible can you please take care of it even for Fedora 33 container too..

Reply:
Thanks. I have changed for the Fedora one as well.